src/amber/dsl/callbacks.cr

```diff
@@ -0,0 +1,21 @@
+module Amber::DSL
+  module Callbacks
+    macro before_action
+      def before_filters
```

Comment:
because I want to trap the body of the macro without running to execute it at a later time, if the `def` would be there the code will execute in the wrong context, in the current case at the class level and not the instance level.